src/Pages/Frontpage/Frontpage.js

```diff
@@ -1,47 +1,52 @@
-import React from "react";
-import Header from "../../Shared/Header";
+import React, { useState } from "react";
 import Survey from "./Survey/Survey";
 import surveyData from "../../Data/survey";
 import SignUpForm from "./SignUpForm";
 import PopupWrapper from "./PopupWrapper";
 
 export default (props) => {
-  // const signUp = () => {
-  //   alert("Sign up function needed");
-  // };
-
-  // const login = () => {
-  //   alert("Login function needed");
-  // };
 
+  const [surveyVisible, setSurveyVisible] = useState(false)
   const surveyClick = () => {
```

Comment:
Transition is not user friendly. When I click 'Test your CO2 footprint' I'm not aware that anything happens, just that I lose footer, so intuitively I scroll down and see the questions.

Reply:
We are aware. Have a resource to do this without manipulating the dom, just haven't gotten around to it.